packages/conventional-commits-parser/src/CommitParser.ts

```diff
@@ -0,0 +1,431 @@
+import type {
+  ParserOptions,
+  ParserRegexes,
+  CommitReference,
+  CommitNote,
+  CommitMeta,
+  Commit
+} from './types.js'
+import { getParserRegexes } from './regex.js'
+import {
+  trimNewLines,
+  appendLine,
+  getCommentFilter,
+  gpgFilter,
+  truncateToScissor
+} from './utils.js'
+import { defaultOptions } from './options.js'
```

Comment:
q, do we need extension here? this file is not present in source, and normally build should take care of that

Reply:
`"moduleResolution": "NodeNext"`
it requires '.js' extension